monkey/monkey_island/cc/next_ui/next.config.js

```diff
@@ -1,11 +1,8 @@
 /** @type {import('next').NextConfig} */
 const nextConfig = {
     reactStrictMode: true,
-    env: {},
-    //output: 'standalone',
     compiler: {
-        removeConsole:
-            process.env.NODE_ENV === process.env.NEXT_PUBLIC_PRODUCTION_KEY
+        removeConsole: false,
```

Comment:
Shouldn't we remove all console logs in production?

Reply:
We probably want to show errors that users can report in case they find any